packages/web/src/components/tipping/tip-audio/TipAudio.module.css

```diff
@@ -259,6 +259,9 @@
 .confirmButton.disabled {
   width: 186px;
```

Comment:
Maybe instead of adding that new class we can just make this wider - looks like it's also supposed to be 251px per figma?

Reply:
made it 252px as that's even and divisible by 4. and yes looks like the figma also update the basic confirm buttpn to be as wide and the error confirm button. good catch thx